packages/client/src/index.ts

```diff
@@ -1,3 +1,7 @@
+import { ErrorCode as WebAPIErrorCode } from '@slack/web-api';
```

Comment:
i prefer having these import statements closer in the file to where the values are used. while that's not generally a preference across all source files, in the case of `index.ts`, it serves as symbol-indirection-map soo its useful.